package.json

```diff
@@ -14,7 +14,7 @@
     "validate": "eslint --ext .js ./src ./env ./server ./test",
     "lintfix": "eslint --ext .js ./src ./env ./server ./test --fix",
 
-    "build": "webpack --progress --profile --config ./env/webpack.prod.config.js",
+    "build": "cross-env ENV_CONFIG=production NODE_ENV=production webpack --progress --profile -p --config ./env/webpack.prod.config.js",
```

Comment:
I guess we not need modify env variables in this task, because ```build``` task became dirty task and build only for production. It will be better split this task in two tasks, ```build``` and ```build:production```
But in your case, I suggest to move ```cross-env ENV_CONFIG=production``` to prepublish task.

PS: ```NODE_ENV=production``` and ```-p``` not needed in this case